src/JUDI.jl

```diff
@@ -50,11 +50,4 @@ end
 # JUDI time modeling
 include("TimeModeling/TimeModeling.jl")
 
-# # Backward compatibility for JUDI
-module TimeModeling
-    using Reexport
-    @reexport using ..JUDI
-    Base.@warn "JUDI.TimeModeling is deprecated, use `using JUDI` instead"
-end
-
```

Comment:
Why do we have to delete this?

Reply:
Because it's deprecated and slows down compilation